pkg/traceql/engine_metrics_test.go

```diff
@@ -364,7 +364,7 @@ func TestQuantileOverTime(t *testing.T) {
 				0,
 			},
 		},
-		`{p="0.5", span.foo="bar"}`: TimeSeries{
+		`{p="0.5", "span.foo"="bar"}`: TimeSeries{
```

Comment:
We can still keep the name label unquoted. The simplest way to do it would be to use a modified label version.Labels.String() without quoting the name

https://github.com/prometheus/prometheus/blob/e32d89af7fb8baee53770de74793e8b2fb828f52/model/labels/labels_common.go#L43-L65